### PR TITLE
Better support for PostGIS

### PR DESCRIPTION
**Pull request by fatkodima**
https://github.com/rgeo/activerecord-postgis-adapter/blob/4f4e28fa3c21f4b31b6bdaea484a3872fe3af252/lib/active_record/connection_adapters/postgis_adapter.rb#L27

### CODE REVIEW

File: lib/active_record_doctor/detectors/base.rb [hunk 1]
```diff
@@ -150,6 +150,10 @@ def models(except: [])
       def underscored_name
         self.class.underscored_name
       end
+
+      def postgresql?
+        connection.adapter_name =~ /postg/i
```
Comment (gregnavis) [inline lib/active_record_doctor/detectors/base.rb:155]: Let's use an array of adapter names that correspond to PostgreSQL. Less matching adapters, higher guarantee those that do match actually work.

**Review by gregnavis — CHANGES_REQUESTED** [review]
Thank you for the contribution. I left one comment.

File: lib/active_record_doctor/detectors/base.rb (hunk 1 above)
Comment (fatkodima) [reply lib/active_record_doctor/detectors/base.rb]: Changed.

Comment (gregnavis) [reply lib/active_record_doctor/detectors/base.rb]: Thank you. I'll merge tomorrow.

**Comment by gregnavis** [comment]
Merged in 92b9479ae00125bf48f8027bb39ffe78e1134551. Thank you, @fatkodima!

**Pull request closed without merging**